game/card.go

```diff
@@ -95,6 +95,17 @@ func newCardHelper(name CardName) *Card {
 			EntersPlayAction: &Action{Type: Play, Card: tokenCard},
 			IsCreature:       true,
 		}
+	case BurningTreeEmissary:
+		manaCard := &Card{
```

Comment:
this `manaCard` will end up named `BurningTreeEmissary`, won't it? hehe

I don't think you want to have a "card" object here. There is no card that is the coming-into-play-ness of a burning tree emissary. But this goes back to the other PR where it didn't seem right to have cards create their own Actions.